docs/src/getting_started.md

```diff
@@ -0,0 +1,146 @@
+# Getting Started with Chmy.jl
+
+[Chmy.jl](https://github.com/PTsolvers/Chmy.jl) is powered by [KernelAbstractions.jl](https://github.com/JuliaGPU/KernelAbstractions.jl) and it is a backend-agnostic toolkit for finite difference computations on multi-dimensional computational staggered grids. In this introductory tutorial, we will show the essence of Chmy.jl in which we resolved a simple 2D diffusion problem. The full code of the tutorial material is available under [diffusion_2d.jl](https://github.com/PTsolvers/Chmy.jl/blob/main/examples/diffusion_2d.jl).
+
+
+## Basic Diffusion
+
+The diffusion equation is a second order parabolic PDE, here for a multivariable function $C(x,t)$ that represents the temperature field showing derivatives in both temporal $\partial t$ and spatial $\partial x$ dimensions, where $D$ is the diffusion coefficient. In 2D we have the following formulation for the diffusion process:
```

Comment:
It's a bit unfortunate that we have `C` as variable name for temperature example where `T` would be more "natural"... Maybe one could change `C` to `T` in the code, or use chemical diffusion with Fick's law (instead of Fourrier's law) in the example?